README.md

```diff
@@ -1,33 +1,25 @@
-# Document AI Bouding Boxes Workshop 
+# Document AI Notebooks 
 
-In this tutorial you will process a document with a Document AI processor and
-visualize its output.
+This repository contains several [Google AI Platform notebooks][notebooks] 
+to be used with the Cloud [Document AI Platform][docai].
 
-![boxes](images/SampleOutput.png)
-![table](images/SampleFormFields.png)
+![boxes](resources/screenshots/SampleFormFields.png)
 
 ## Prerequisites 
 
-You must be familar GCP and have your own project created.
+You must be familar GCP and have your own project with billing enabled.
```

Comment:
Maybe modify it to:

Working knowledge of the following Google Cloud Platform products:
- [Google Cloud Storage](https://cloud.google.com/storage)
- [AI Platform Notebooks](https://cloud.google.com/ai-platform-notebooks) (Optional but would simplify the initial setup)
- Basic Understanding of [Google Document AI Concepts](https://cloud.google.com/document-ai/docs/basics) and [Processors](https://cloud.google.com/document-ai/docs/processor-overview)